### PR TITLE
Seperate RenderItemChangeY into its own mixin and disable it when canvas is present.

### LINKED ISSUE #10: Incompatability with newer Canvas builds.
Any canvas build newer than 1.0.1017 crashes with slight gui modifications due to a mixin inject error. [Here is the log.](https://gist.github.com/loper12/b37ee3ff2d0cabc5175f2e3369a2a037) This seems to be caused by [this injection in slight gui modifications](https://github.com/shedaniel/slight-gui-modifications/blob/8816c7a5c379ec25dbd50ea5feec53095c742a2e/src/main/java/me/shedaniel/slightguimodifications/mixin/MixinItemRenderer.java#L46), and [this overwrite in canvas.](https://github.com/grondag/canvas/blob/9817ce5d30b42b4d52fe95bdda1982f7160751ba/src/main/java/grondag/canvas/mixin/MixinItemRenderer.java#L42) This issue is being tracked at grondag/canvas#126

### PR DESCRIPTION
**Pull request by Dolphin2Point1**
Fixes #10. All this does is disable RenderItemChangeY when canvas is present. This isn't the best solution, but the better solution would probably be along the lines of adding canvas compatibility.

### CODE REVIEW

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
   },
   "accessWidener": "slight-gui-modifications.aw",
   "mixins": [
-    "slight-gui-modifications.mixins.json"
+    "slight-gui-modifications.mixins.json",
+    "item-renderer.mixins.json"
```
Comment (shedaniel) [inline src/main/resources/fabric.mod.json]: can you prefix this with the mod id

**Review by shedaniel — CHANGES_REQUESTED** [review]
looks fine apart from that small change

File: src/main/resources/fabric.mod.json (hunk 1 above)
Comment (Dolphin2Point1) [reply src/main/resources/fabric.mod.json]: ok

**Comment by Dolphin2Point1** [comment]
I've realized that I can clean this all into the one plugin, which may make it easier to disable other mixins when you experience incompatibilities. Will do this, unless you'd like to object.

**Comment by shedaniel** [comment]
It is fine

**Comment by Dolphin2Point1** [comment]
Fine for me to add it? or fine as it is? I'd be completely fine to add it, but if you'd like me to leave it as it is, it is ok.

**Comment by shedaniel** [comment]
It is fine for you to add it

**Comment by Dolphin2Point1** [comment]
Done.

**Review by shedaniel — APPROVED** [review]

**Pull request merged**